app/scripts/lib/rpc-method-middleware/handlers/index.js

```diff
@@ -7,6 +7,15 @@ import sendMetadata from './send-metadata';
 import switchEthereumChain from './switch-ethereum-chain';
 import watchAsset from './watch-asset';
 
+///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+import mmiSupported from './institutional/mmi-supported';
+import mmiAuthenticate from './institutional/mmi-authenticate';
+import mmiPortfolio from './institutional/mmi-portfolio';
+import mmiOpenSwaps from './institutional/mmi-open-swaps';
+import mmiCheckIfTokenIsPresent from './institutional/mmi-check-if-token-is-present';
+import mmiSetAccountAndNetwork from './institutional/mmi-set-account-and-network';
```

Comment:
we have one missing no? 🤔 
`mmiOpenAddHardwareWallet` from  `mmi-open-add-hardware-wallet`